code_share/app_code_share/templates/app_code_share/homepage.html

```diff
@@ -32,27 +35,102 @@
                         {%else%}
                          unique Id:<h4>{{code_share.hash_value}}</h4>
                         {%endif%}
-                        <textarea name="code_snippet">{{code_share.code}}</textarea><br>
-                        <input type="submit" value="edit">
+                        <textarea id="code_snippet" name="code_snippet">{{code_share.code}}</textarea><br>
+                        <input id="submit_edit" type="submit" value="edit">
                         <br>          
                          <a href="{% url 'code_share:app_home'  %}" >
-                             <button id="create" >create new </button>
+                             <div id="create" >create new </div>
                         </a><br>          
                      {%else%}
-                        File Name:
-                        <input type="text" id="file_field" name="file_name" placeholder="Not neccessary , but dont leave spaces in between the name." value="{{code_share.file_name}}">
-                        <textarea name="code_snippet">{{code_share.code}}</textarea><br>
-                        <input type="submit" value="submit"><br> 
+                        <input type="text" id="file_field" name="file_name" placeholder="File Name" value="{{code_share.file_name}}">
+                        
+                        <textarea id="code_snippet" name="code_snippet" placeholder="Code Snippet">{{code_share.code}}</textarea><br>
+                        <input id="submit_new" type="submit" value="submit" onmouseover="check_content(this)"><br> 
                      {%endif%}
                     </form>
                     <blockquote>
                         <p><em>Share your shitty code here. nothing is necessary. Put code and submit.BAM.</em></p>
-                        <p><em>You can access your code via file name or the unique number. Better to add file name to acces it later.</em></p>
                     </blockquote>
+
                 </div>
             </div>
         </div>
     </div>
-    
+
+    {% include "fork_github.html" %}
+
   </body>
-</html>
\ No newline at end of file
+
+  <script
+  src="https://code.jquery.com/jquery-3.2.1.min.js"
+  integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4="
+  crossorigin="anonymous"></script>
```

Comment:
jQuery is required for event handler on submit button to work.
It does work on button without jQuery but not on Textarea, dunno why.